rules/style.js

```diff
@@ -0,0 +1,26 @@
+const isDev = process.env.NODE_ENV === 'development';
```

Comment:
Rename this directory to config since these are not the actual rules but parts of the config.

Reply:
I disagree. Name `rules` is inspired by the popular [airbnb](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) `eslint` configs. And these are actually rules, as they are imported in the `index.js`, as rules. 

[`eslint` documentatio](https://eslint.org/docs/6.0.0/developer-guide/shareable-configs#local-config-file-resolution)n suggests naming this folder `lib`, but I like the airbnb convention more, because it makes sense.`lib` and `config` are descriptive enough.

I'll leave it as it is.